rpcs3/Emu/RSX/D3D12/D3D12GSRender.cpp

```diff
@@ -166,6 +167,7 @@ D3D12GSRender::D3D12GSRender()
 		// Adapter with specified name
 		if (adapter_name == desc.Description)
 		{
+			vram_size = desc.DedicatedVideoMemory != 0 ? desc.DedicatedVideoMemory : desc.DedicatedSystemMemory;
```

Comment:
Well, yes, thats how IGP vram is allocated. Some bioses allow users to set the value in the bios so that the IGP sees it. Dedicated vram on IGPs should report 0 (It used to in the past anyway, dx9)